effort_joint_trajectory_controller:
  type: effort_controllers/JointTrajectoryController
  joints:
    - panda_joint1
    - panda_joint2
    - panda_joint3
    - panda_joint4
    - panda_joint5
    - panda_joint6
    - panda_joint7

  gains:
    panda_joint1: {p: 300, i: 0, d: 10}
    panda_joint2: {p: 300, i: 0, d: 10}
    panda_joint3: {p: 300, i: 0, d: 10}
    panda_joint4: {p: 300, i: 0, d: 10}
    panda_joint5: {p: 300, i: 0, d: 10}
    panda_joint6: {p: 300, i: 0, d: 10}
    panda_joint7: {p: 300, i: 0, d: 10}

  constraints:
    goal_time: 0.6
    stopped_velocity_tolerance: 0.05
    panda_joint1: {trajectory: 0.1, goal: 0.1}
    panda_joint2: {trajectory: 0.1, goal: 0.1}
    panda_joint3: {trajectory: 0.1, goal: 0.1}
    panda_joint4: {trajectory: 0.1, goal: 0.1}
    panda_joint5: {trajectory: 0.1, goal: 0.1}
    panda_joint6: {trajectory: 0.1, goal: 0.1}
    panda_joint7: {trajectory: 0.1, goal: 0.1}

